src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,32 @@
+import Image from "next/image"
+import BannerImage from "@/public/images/banner.png"
+import { Button } from "../shared/Button"
+
+import styles from "./Banner.module.scss"
+
+export default function Banner() {
+    return (
+        <section className={styles.container}>
+            <Image className={styles.image} src={BannerImage} alt=""></Image>

```

Comment:
```suggestion
            <Image className={styles.image} src={BannerImage} alt="" />

```
У каждого изображения должен быть alt.